test/dist_autograd_test.py

```diff
@@ -744,28 +721,19 @@ def test_context_cleanup_tensor_no_grad(self):
         self.assertTrue(success)
 
     @dist_init
-    def test_context_cleanup_no_tensors(self):
-        self._initialize_pg()
+    def test_context_cleanup_tensor_with_grad(self):
+        t1 = torch.ones(3, 3, requires_grad=True)
+        t2 = torch.zeros(3, 3, requires_grad=True)
+        self.context_cleanup_test_helper(rpcargs=(t1, t2), func=torch.add)
```

Comment:
Could you change this to not have a `t2` at all and just use `rpc_args=(t1, t1)`?

Reply:
The test seems to fail a couple of times when I did that. I'm not sure if it matters that `t1` is `ones` and `t2` is `zeros`.

Reply:
At first glance this shouldn't happen but shouldn't block this PR either. I will investigate more later.